Makefile

```diff
@@ -36,16 +36,19 @@ endif
 IMG_SUFFIX ?= -$(subst _,,$(shell whoami))
 
 REGISTRY           ?= docker.elastic.co
+DOCKERHUB_REGISTRY = docker.io
 REGISTRY_NAMESPACE ?= eck-dev
 NAME               ?= eck-operator
+ORGANIZATION       ?= elastic
 SNAPSHOT           ?= true
 VERSION            ?= $(shell cat VERSION)
 TAG                ?= $(shell git rev-parse --short=8 --verify HEAD)
 IMG_NAME           ?= $(NAME)$(IMG_SUFFIX)
 IMG_VERSION        ?= $(VERSION)-$(TAG)
 
-BASE_IMG       := $(REGISTRY)/$(REGISTRY_NAMESPACE)/$(IMG_NAME)
-OPERATOR_IMAGE ?= $(BASE_IMG):$(IMG_VERSION)
+BASE_IMG                 := $(REGISTRY)/$(REGISTRY_NAMESPACE)/$(IMG_NAME)
+OPERATOR_IMAGE           ?= $(BASE_IMG):$(IMG_VERSION)
+OPERATOR_DOCKERHUB_IMAGE ?= $(DOCKERHUB_REGISTRY)/$(ORGANIZATION)/$(IMG_NAME):$(IMG_VERSION)
```

Comment:
Nit: Since we already have a lot of variables, I'd be tempted to add just the minimum.

```diff
-DOCKERHUB_REGISTRY = docker.io
 REGISTRY_NAMESPACE ?= eck-dev
 NAME               ?= eck-operator
-ORGANIZATION       ?= elastic
 SNAPSHOT           ?= true
 VERSION            ?= $(shell cat VERSION)
 TAG                ?= $(shell git rev-parse --short=8 --verify HEAD)
@@ -48,7 +46,7 @@ IMG_VERSION        ?= $(VERSION)-$(TAG)
 
 BASE_IMG                 := $(REGISTRY)/$(REGISTRY_NAMESPACE)/$(IMG_NAME)
 OPERATOR_IMAGE           ?= $(BASE_IMG):$(IMG_VERSION)
-OPERATOR_DOCKERHUB_IMAGE ?= $(DOCKERHUB_REGISTRY)/$(ORGANIZATION)/$(IMG_NAME):$(IMG_VERSION)
+OPERATOR_DOCKERHUB_IMAGE ?= docker.io/elastic/$(IMG_NAME):$(IMG_VERSION)
```

Reply:
yeah, I can get behind that.